main.py

```diff
@@ -202,7 +204,8 @@ def main(args):
     )
 
     data_loader_val = torch.utils.data.DataLoader(
-        dataset_val, batch_size=int(1.5 * args.batch_size),
+        dataset_val, sampler=sampler_val,
+        batch_size=int(1.5 * args.batch_size),
         shuffle=False, num_workers=args.num_workers,
```

Comment:
Can you also remove `shuffle=False`, so that it's clear that all shuffling is handled by the sampler?